README.adoc

```diff
@@ -28,6 +35,14 @@ This plugin is published under the Apache License, version 2.0.
 
 == ChangeLog
 
+=== gitbucket-asciidoctor-plugin 0.1.2 - 2016-01-30
```

Comment:
I'm ok with a prefilled Changelog, but the release date isn't correct. At least, not for this repo, as I obviously have never release these versions.